common/tools/dev-tool/src/commands/run/build-test.ts

```diff
@@ -172,7 +172,7 @@ async function compileForEnvironment(
   }
 
   // Check if the TypeScript config uses package name imports or "internal" paths
-  const shouldSkipOverrides = hasPackageOrInternalPaths(tsConfigJSON);
+  const shouldSkipOverrides = false; //hasPackageOrInternalPaths(tsConfigJSON);
```

Comment:
This is needed because we still need to build the src directory for all libraries browser tests. It will be switched off once all tests are migrated to import from the package name and not from the src folder.